oidc-controller/api/authSessions/crud.py

```diff
@@ -43,19 +43,23 @@ async def get(self, auth_session_id: str) -> AuthSession:
 
         return AuthSession(**auth_sess)
 
-    async def patch(self, auth_session_id: str, data: AuthSessionPatch) -> AuthSession:
+    async def patch(self, id: str, data: AuthSessionPatch) -> AuthSession:
```

Comment:
Should we check whether the provided id is valid in other CRUD methods as well other that `post`?

Reply:
Ok, i figured out why this is. 

The patch method called by the acapy-webhook handler passes in `id` as a bson.objectid which is a superclass of the PyObjectId that is being used.... but the get method is being passed `id` as a string, so i needed to convert it 

I can leave it and leave a comment as to why this is? or I can cast it to the str in the webhook handler and convert it to a PyObjectId in the CRUD method for consistency. 

I think i'd prefer the second. Thoughts?

Reply:
I agree with the second approach for consistency